cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/SelfAwareJoinDecider.scala

```diff
@@ -34,10 +34,13 @@ import akka.event.Logging
    * during [[akka.management.scaladsl.AkkaManagement.start()]], hence we accept blocking on
    * this initialization.
    */
-  private[bootstrap] def selfContactPoint: (String, Int) =
-    Try(Await.result(ClusterBootstrap(system).selfContactPoint, 10.seconds)).getOrElse(throw new IllegalStateException(
+  private[bootstrap] def selfContactPoint: (String, Int) = {
```

Comment:
I'd say a `lazy val` would be a good match here instead of re-evaluating the `Await` block, but I don't know if `lazy vals` are kinda looked down upon here. Poor lazy vals.